static/templates/compose_all_everyone.handlebars

```diff
@@ -1,6 +1,10 @@
 <div class="compose-all-everyone">
     <span class="compose-all-everyone-msg">
-        {{#tr this}}Are you sure you want to mention all <strong>__count__</strong> people in this stream?{{/tr}}
+        {{#if is_announce_stream}}
```

Comment:
Let's not use a boolean here.  Just make it a separate type of message.

Reply:
OK, I made it a separate message in 41863ae.